include/ceed/backend.h

```diff
@@ -219,7 +219,7 @@ CEED_EXTERN int CeedGetDelegate(Ceed ceed, Ceed *delegate);
 CEED_EXTERN int CeedSetDelegate(Ceed ceed, Ceed delegate);
 CEED_EXTERN int CeedGetObjectDelegate(Ceed ceed, Ceed *delegate, const char *obj_name);
 CEED_EXTERN int CeedSetObjectDelegate(Ceed ceed, Ceed delegate, const char *obj_name);
-CEED_EXTERN int CeedGetOperatorfallback_resource(Ceed ceed, const char **resource);
+CEED_EXTERN int CeedGetOperatorFallbackResource(Ceed ceed, const char **resource);
```

Comment:
Let me know if this change was intentional, but it looks to me like a find and replace bug so I reverted it.

Reply:
Crap, yeah. That was a goof on my part